app/lib/efile/md/md502_calculator.rb

```diff
@@ -361,12 +376,12 @@ def calculate_deduction_method
         s_mfs_d: {
           12000 => 1_800,
           17999 => ->(x) { x * 0.15 },
-          18000 => 2_700,
+          Float::INFINITY => 2_700,
```

Comment:
@jenny-heath helped sort out there was a bug here where its supposed to be "$18,000 or over" and so the band is changed in this line. Here is the original [ticket](https://codeforamerica.atlassian.net/browse/FYST-758) for reference.